vmd-server/src/vmd/kvm_util.rs

```diff
@@ -1,10 +1,11 @@
+#![allow(dead_code)]
 use kvm_ioctls::{
     Kvm, Cap
 };
 use thiserror::Error;
 
 #[derive(Debug, Error)]
-pub enum VdmError {
+pub enum KvmError {
```

Comment:
For completeness and coming design documentent change on no networking on host, we likely have to drop direct KVM API support as `vmd` won't be running on host but guest VM instead. No actions on dropping the KVM API dependency required for now.

Reply:
Noted. Won't be a big issue down the line. I haven't invested too much time into the kvm stuff.